### DISCUSSION TITLE
How do I update ComfyUI

### DISCUSSION THREAD

**Discussion by CaBaccoRic** [comfyanonymous/ComfyUI#1425] [Q&A]
Hi everyone! I installed Comfy as soon as it came out, but now after the summer break, i seem to have lost some new features like the manager and the control net. Before installing/downloading them i think i need an update, is there a code to update ComfyUI rather than re-install it?

**Comment by comfyanonymous** [comment]
You can update it with: update/update_comfyui.bat on the standalone or "git pull".

If you have an old standalone from back in march I recommend downloading the latest one.

**Reply by derpmagician** [reply]
thanks, i will do that

**Comment by aiworksnew1** [comment]
how to update comfyui in colab pro to latest version of comfyui

**Reply by ltdrdata** [reply]
Check the `UPDATE_COMFY_UI` checkbox and execute.
